axios/axios
# Experimental http2 adapter

###  Experimental HTTP2 Adapter
* This is exploratory, hopefully we can build a real adapter from this, I'll keep adding bits and pieces but I was able to test an http2 request using axios with this, it's very barebones and needs clean up  and plenty of more functionality before being ready for an alpha
* Related to https://github.com/axios/axios/issues/1175
* To test HTTP2 requests you'll need to:
  * Run `openssl req -x509 -newkey rsa:2048 -nodes -sha256 -subj '/CN=localhost' \
  -keyout localhost-privkey.pem -out localhost-cert.pem` which will generate the two files needed to run `http2_test.js`
  * After running that command you should see `localhost-privkey.pem` and `localhost-cert.pem` in your folder
  * In a terminal tab run `server.js` and in another tab run `http2_test.js` which will immediately send a request to the server

Feel free to comment on this PR, feedback is welcome
